src/ontology/cl-edit.owl

```diff
@@ -27702,6 +27707,29 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P32848))
 
 
+# Class: obo:CL_4023051 (vascular leptomeningeal like cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "ILX:0770143") obo:IAO_0000115 obo:CL_4023051 "A neurohypophyseal cell that expresses Pdgfra and Lum. They are fibroblast-esque cells that are attached to vessels, located in the space betwee smooth muscle and astrocytes. Vascular leptomeningeal cells (VLMCs) like cells are a family of functionally related (but organ-specific) mesothelial fibroblasts that form protective membranes around internal organs, including the pia and arachnoid membranes of the brain.")
```

Comment:
This definition looks  much more specific & detailed than necessary.  I'm not sure we need the markers at all to define it.  How about:
"A type of mesothelial fibroblast that is derived from the neural crest and is a key component of the pia and arachnoid membranes surrounding the brain."

Add: 
subClassOf part_of  some UBERON_0000391 # leptomeninx

Reply:
(BTW - interested to know why this is being called a 'neurohypophyseal cell' and what that might mean for functional assertions, & other potential axes of classification.  Would need a paper with evidence to back it up though.)

Reply:
Maybe chase down VLMC refs for makers from https://www.nature.com/articles/s41586-018-0654-5  - could have some other details.